diff --git a/inferno_aoip/src/device_server/flows_tx.rs b/inferno_aoip/src/device_server/flows_tx.rs
index bfc8c38..569ef43 100644
--- a/inferno_aoip/src/device_server/flows_tx.rs
+++ b/inferno_aoip/src/device_server/flows_tx.rs
@@ -167,7 +167,7 @@ impl<P: ProxyToSamplesBuffer> FlowsTransmitterInternal<P> {
             let samples = &tmp_samples[0..flow.fpp];
             match flow.bytes_per_sample {
               2 => write_s16_samples(samples, &mut pbuff, start, stride, Some(dither_rng)),
-              3 => write_s24_samples(samples, &mut pbuff, start, stride, Some(dither_rng)),
+              3 => write_s24_samples::<_, SmallRng>(samples, &mut pbuff, start, stride, None), // pi9696 test instrument: no dither
               4 => write_s32_samples::<_, SmallRng>(samples, &mut pbuff, start, stride, None),
               other => {
                 error!("BUG: unsupported bytes per sample {}", other);
